website/dynamic-blocks/do-and-avoid.js

```diff
@@ -27,6 +27,34 @@ const UPLOAD_IMAGE = gql`
 	}
 `;
 
+const Figure = (props) => (
+	<figure
+		css={{
+			margin: 0,
+			flexBasis: '50%',
+			display: 'flex',
+			flexDirection: 'column',
+			alignItems: 'start',
+			justifyContent: 'flex-start',
+		}}
+		{...props}
+	/>
+);
+
+const Image = (props) => {
+	const mq = useMediaQuery();
+	return (
+		<img
+			css={mq({
+				maxWidth: '100%',
+				width: '100%',
```

Comment:
Responsive image? Should prob also have `height: 'auto'`. 

Are you forcing it to be as wide as its container or as wide as it can be (without enlarging and pixelating)? Shouldn't need both width and maxWidth.